### PR TITLE
remove and replace invalid links

### PR DESCRIPTION
**Pull request by awesome-doge**
* Add "Assume UTXO" video link
* Fixed invalid link "Taproot: SegWit version 1 output spending rules"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Security Models|Neutrino |[BIP 157](https://github.com/bitcoin/bips/blob/master/
 Security Models|Committed bloom filters|[Committed bloom filters for improved wallet performance and SPV security](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-May/012636.html)|
 Security Models|Fraud proofs|[Fraud Proofs (2018)](https://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/fraud-proofs/), [Fraud Proofs: Improving the ability of SPV clients to detect invalid chains (gist)](https://gist.github.com/justusranvier/451616fa4697b5f25f60), [Bitcoin Wizards Fraud Proof thread](https://people.xiph.org/~greg/bitcoin-wizards-fraud-proof.log.txt), [Improving SPV security with PoW fraud proofs](https://www.mail-archive.com/bitcoin-dev@lists.linuxfoundation.org/msg07913.html)|
 Security Models|Committed UTXO hashes|[UTXO set commitment hash](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-October/011638.html)|
-Security Models|Assume UTXO|[Assume UTXO FAQs](https://github.com/jamesob/assumeutxo-docs/tree/2019-04-proposal/proposal)|
+Security Models|Assume UTXO|[Assume UTXO FAQs](https://github.com/jamesob/assumeutxo-docs/tree/2019-04-proposal/proposal)[Assume UTXO (video)](https://youtu.be/PoEoG6sP1hw)|
```
Comment (adamjonas) [inline README.md:45]:
```suggestion
Security Models|Assume UTXO|[Assume UTXO FAQs](https://github.com/jamesob/assumeutxo-docs/tree/2019-04-proposal/proposal), [Assume UTXO (video)](https://youtu.be/PoEoG6sP1hw)|
```

**Comment by adamjonas** [comment]
Thanks, @awesome-doge! Looks good. Just a formatting nit and I'll merge it.

**Comment by awesome-doge** [comment]
* Add fix "User Activated Soft Forks: the BIP148 alternative" invalid link

**Comment by awesome-doge** [comment]
@adamjonas I unexpectedly found more invalid links. So I modified it by the way.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Security Models|Overview|[Security models seminar video](https://youtu.be/6gGcS4
...
 Security Models|Defining SPV and lightclients|[Light Clients](https://gist.github.com/ariard/1034cd7624805d53334e80d4712fb8ee)|
 Security Models|BIP 37 (bloom filters)|[BIP 37](https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki), [On the Privacy Provisions of Bloom Filters in Lightweight Bitcoin Clients](https://eprint.iacr.org/2014/763.pdf)|
-Security Models|Neutrino |[BIP 157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki), [Neutrino: The Lighter Side of Lightning](https://blog.lightning.engineering/posts/2018/10/17/neutrino.html), [Index for BIP 157 block filters](https://github.com/bitcoin/bitcoin/pull/14121), [Exploring Neutrino](https://vimeo.com/316626387)|
+Security Models|Neutrino |[BIP 157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki), [Neutrino: The Lighter Side of Lightning](https://blog.lightning.engineering/posts/2018/10/17/neutrino.html), [Index for BIP 157 block filters](https://github.com/bitcoin/bitcoin/pull/14121)|
```
Comment (adamjonas) [inline README.md:41]: this was moved to youtube: https://www.youtube.com/watch?v=luRGJc3uIIo

File: README.md [hunk 3]
```diff
@@ -123,7 +123,7 @@ Wallet|Fees|[The Fee Market Explained](https://medium.com/@jimmysong/the-fee-mar
 Wallet|Fee estimation|[Fee estimation code exercise](https://youtu.be/PYA1f2xlIOM)|
 Wallet|Replace by Fee|[Opt-in Full Replace-by-Fee Signaling](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki), [Support for zero-confirmation transactions at Bitcoin ATM](https://coinatmradar.com/blog/support-zero-confirmation-transactions-at-bitcoin-atm/)|
 Wallet|Coin selection| [Unspent Management and Coin Selection](https://www.youtube.com/watch?v=hrlNN3BSB6w), [Coin Selection](https://bitcoinedge.org/tutorial/dpp105-karl-johan-alm-bitcoin-wallets-coin-selection), [Coin Selection (Scaling 2016)](https://scalingbitcoin.org/transcript/milan2016/coin-selection), [An Evaluation of Coin Selection Strategies](http://murch.one/wp-content/uploads/2016/11/erhardt2016coinselection.pdf)|
-Wallet|Hardware wallets with Bitcoin core|[Using your hardware wallet with Bitcoin Core](https://vimeo.com/316634495)|
+Wallet|Hardware wallets with Bitcoin core| |
```
Comment (adamjonas) [inline README.md:126]: this was moved to youtube: https://www.youtube.com/watch?v=hWgSjtbyN8U

File: README.md [hunk 4]
```diff
@@ -138,23 +138,23 @@ Scripts & Contracts|ZK STARKS|[Scalable, transparent, and post-quantum secure co
 Scripts & Contracts|Discreet Log Contracts|[Discreet Log Contracts](https://dci.mit.edu/research/smart-contracts-discrete-log-contracts), [Discreet Log Contracts: invisible smart contracts on the Bitcoin blockchain](https://medium.com/@gertjaap/discreet-log-contracts-invisible-smart-contracts-on-the-bitcoin-blockchain-cc8afbdbf0db)|
 Scripts & Contracts|Miniscript| [Miniscript](http://diyhpl.us/wiki/transcripts/stanford-blockchain-conference/2019/miniscript/), [Policy to Miniscript compiler](http://bitcoin.sipa.be/miniscript/)|
 Scripts & Contracts|State of script|[The State of Script (2018)](https://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/bitcoin-script/)|
-Fungibility & Scalability| Overview |[The current state of Bitcoin fungibility (2019)](https://vimeo.com/316635787)|
+Fungibility & Scalability| Overview | |
```
Comment (adamjonas) [inline README.md:141]: also moved to youtube: https://www.youtube.com/watch?v=lz1wB_hWpvo

File: README.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ Cryptography|Bitcoin, Chance and Randomness|[Bitcoin, Chance and Randomness](htt
...
 Cryptography|zero knowledge proofs|[Introduction to SNARKs](https://www.youtube.com/watch?v=jr95o_k_SwI&feature=youtu.be)|
-Cryptography|Bulletproofs|[How Bulletproofs Could Make Bitcoin Privacy Less Costly](https://bitcoinmagazine.com/articles/how-bulletproofs-could-make-bitcoin-privacy-less-costly/), [Bulletproofs: Faster Rangeproofs and Much More](https://blockstream.com/2018/02/21/bulletproofs-faster-rangeproofs-and-much-more/), [Bulletpoints on bulletproofs](https://joinmarket.me/blog/blog/bulletpoints-on-bulletproofs/), [From Zero Knowledge Proofs to Bulletproofs Paper](https://joinmarket.me/blog/blog/from-zero-knowledge-proofs-to-bulletproofs-paper/), [Building on Bulletproofs](https://medium.com/@cathieyun/building-on-bulletproofs-2faa58af0ba8)|
+Cryptography|Bulletproofs|[How Bulletproofs Could Make Bitcoin Privacy Less Costly](https://bitcoinmagazine.com/articles/how-bulletproofs-could-make-bitcoin-privacy-less-costly/), [Bulletproofs: Faster Rangeproofs and Much More](https://blockstream.com/2018/02/21/bulletproofs-faster-rangeproofs-and-much-more/),  [Building on Bulletproofs](https://medium.com/@cathieyun/building-on-bulletproofs-2faa58af0ba8)|
```
Comment (adamjonas) [inline README.md:85]: such a shame that waxwing's blog is down. maybe substitute https://medium.com/@nopara73/confidential-transactions-bulletproofs-the-elephant-in-the-room-cfdb37ce509?

File: README.md [hunk 6]
```diff
@@ -32,17 +32,17 @@ Soft Forks & protocol overview|Fork categorization (soft, hard, evil, etc.)|[Bet
...
+Security Models|Neutrino |[BIP 157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki), [Neutrino: The Lighter Side of Lightning](https://blog.lightning.engineering/posts/2018/10/17/neutrino.html), [Index for BIP 157 block filters](https://github.com/bitcoin/bitcoin/pull/14121)|
 Security Models|Committed bloom filters|[Committed bloom filters for improved wallet performance and SPV security](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-May/012636.html)|
 Security Models|Fraud proofs|[Fraud Proofs (2018)](https://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/fraud-proofs/), [Fraud Proofs: Improving the ability of SPV clients to detect invalid chains (gist)](https://gist.github.com/justusranvier/451616fa4697b5f25f60), [Bitcoin Wizards Fraud Proof thread](https://people.xiph.org/~greg/bitcoin-wizards-fraud-proof.log.txt), [Improving SPV security with PoW fraud proofs](https://www.mail-archive.com/bitcoin-dev@lists.linuxfoundation.org/msg07913.html)|
```
Comment (adamjonas) [inline README.md:43]: Greg Maxwell's server appears down. We'll have to pull out this too.

Comment (awesome-doge) [reply README.md]: okay, got it.
So I removed all links related to "https://people.xiph.org"

**Comment by adamjonas** [comment]
What an awesome contribution, @awesome-doge!

**Pull request merged**